core/primitives/src/runtime/parameter_table.rs

```diff
@@ -22,6 +22,7 @@ pub(crate) struct ParameterTable {
 /// Error returned by ParameterTable::from_txt() that parses a runtime
 /// configuration TXT file.
 #[derive(Error, Debug)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
I think it is okay if you derived `PartialEq` unconditionally here. Another useful derive could be `Clone`.

Reply:
Adding in the strum errors as source counteracts that, sadly. I ended up removing `PartialEq` entirely and instead use `assert_machtes!`.